lib/properties/menage-properties.ts

```diff
@@ -39,12 +39,12 @@ export default {
       { value: 12, label: "Oui" },
       {
         value: -2,
-        label: "Non, j'ai emmenagé il y a moins de 3 mois",
+        label: "Non, j'ai emménagé il y a moins de 3 mois",
         isRelevant: ({ simulation }) => {
           const value = getAnswer(
             simulation.answers.current,
             "menage",
-            "statut_occupation_logement"
+            "_logementType"
           )
           return value?.startsWith("locataire")
```

Comment:
Pour être plus explicite, maintenant que c'est possible avec les 2 variables, j'aurais une préférence pour remplacer :
```suggestion
          return _logementType ==="locataire"
```

Reply:
J'ai fait la même remarque à Valérie à l'instant! Je corrige, merci 👍